packages/std/src/ado_base/rates.rs

```diff
@@ -127,7 +127,7 @@ impl LocalRateValue {
 #[cw_serde]
 pub struct LocalRate {
     pub rate_type: LocalRateType,
-    pub recipients: Vec<Recipient>,
+    pub recipient: Recipient,
```

Comment:
This is state breaking, will need a migration handler.

Reply:
Will the latest commit do for now?